migrator/migrator_it_test.go

```diff
@@ -1000,101 +1110,1843 @@ func testMigratorStats(t *testing.T) {
+		if err := mig.dmIN.SetFilter(filter); err != nil {
+			t.Error("Error when setting Filter ", err.Error())
+		}
+		if err := mig.dmIN.SetThresholdProfile(thp); err != nil {
+			t.Error("Error when setting threshold ", err.Error())
+		}
+		if err := mig.dmIN.SetThreshold(th); err != nil {
+			t.Error("Error when setting threshold ", err.Error())
+		}
+		err := mig.oldDataDB.setV2ActionTrigger(threshold)
+		if err != nil {
+			t.Error("Error when setting threshold ", err.Error())
+		}
+		currentVersion := engine.CurrentDataDBVersions()
+		currentVersion[utils.Thresholds] = 1
+		err = mig.dmOut.DataDB().SetVersions(currentVersion, false)
+		if err != nil {
+			t.Error("Error when setting version for threshold ", err.Error())
+		}
+		err, _ = mig.Migrate([]string{utils.MetaThresholds})
+		if err != nil {
+			t.Error("Error when migrating threshold ", err.Error())
+		}
+		result, err := mig.dmOut.GetThreshold("cgrates.org", threshold.ID, true, utils.NonTransactional)
+		if err != nil {
+			t.Error("Error when getting threshold ", err.Error())
+		}
+		if !reflect.DeepEqual(th, result) {
+			t.Errorf("Expecting: %+v, received: %+v", th, result)
+		}
+		thpr, err := mig.dmOut.GetThresholdProfile(thp.Tenant, thp.ID, true, utils.NonTransactional)
+		if err != nil {
+			t.Error("Error when getting thresholdProfile ", err.Error())
+		}
+		if !reflect.DeepEqual(thp.ID, thpr.ID) {
+			t.Errorf("Expecting: %+v, received: %+v", thp.ID, thpr.ID)
+		}
+	case dbtype == utils.MONGO:
+		if err := mig.dmIN.SetFilter(filter); err != nil {
+			t.Error("Error when setting Filter ", err.Error())
+		}
+		if err := mig.dmIN.SetThresholdProfile(thp); err != nil {
+			t.Error("Error when setting Threshold ", err.Error())
+		}
+		if err := mig.dmIN.SetThreshold(th); err != nil {
+			t.Error("Error when setting Threshold ", err.Error())
+		}
+		err := mig.oldDataDB.setV2ActionTrigger(threshold)
+		if err != nil {
+			t.Error("Error when setting Threshold ", err.Error())
+		}
+		currentVersion := engine.CurrentDataDBVersions()
+		currentVersion[utils.Thresholds] = 1
+		err = mig.dmOut.DataDB().SetVersions(currentVersion, false)
+		if err != nil {
+			t.Error("Error when setting version for Threshold ", err.Error())
+		}
+		err, _ = mig.Migrate([]string{utils.MetaThresholds})
+		if err != nil {
+			t.Error("Error when migrating Threshold ", err.Error())
+		}
+		result, err := mig.dmOut.GetThreshold("cgrates.org", threshold.ID, true, utils.NonTransactional)
+		if err != nil {
+			t.Error("Error when getting Threshold ", err.Error())
+		}
+		if !reflect.DeepEqual(th, result) {
+			t.Errorf("Expecting: %+v, received: %+v", th, result)
+		}
+		thpr, err := mig.dmOut.GetThresholdProfile(thp.Tenant, thp.ID, true, utils.NonTransactional)
+		if err != nil {
+			t.Error("Error when getting thresholdProfile ", err.Error())
+		}
+		if !reflect.DeepEqual(thp.ID, thpr.ID) {
+			t.Errorf("Expecting: %+v, received: %+v", thp.ID, thpr.ID)
+		}
+	case dbtype == Move:
```

Comment:
dbtype == Move needs review since it does not look too logic here. It should be more like migrator action = move